docs/manual/installation/update-contao.en.md

```diff
@@ -39,9 +39,9 @@ when writing these lines is **version 4.10.4**.
 
 ### Long-Term Support Versions
 
-With version 2.11, the [release cycle of Contao](https://contao.org/de/release-plan.html) has been adjusted and 
-Long-Term Support (LTS) versions have been introduced that are supported and updated for 24 months, even if newer Contao 
-versions have been released in the meantime. An overview of all Contao versions is available on [Wikipedia](https://en.wikipedia.org/wiki/Contao#Versions).
+The [release cycle](https://contao.org/de/release-plan.html) of Contao also includes a Long-Term Support. Contao LTS versions are 
+supported and updated for up to 4 years, even if newer Contao versions have been released in the meantime. An overview of all Contao versions 
+can be found on [Wikipedia](https://de.wikipedia.org/wiki/Contao#Versionen).
```

Comment:
May be we could mention that the LTS gets a total 3 years of active support and 1 year of security only support (no non-security related bug fixes).

Reply:
See Bjarke review.